### PR TITLE
Release 6.0.0

### PR DESCRIPTION
**Pull request by tonidero**
**RevenueCat Capacitor SDK v6** is here!! 😻

## Important Announcement: Official Ownership Transition
We are thrilled to announce that RevenueCat has officially taken over the ownership and development of this plugin from @riderx.
This transition marks the beginning of a new chapter for Capacitor Purchases, as it becomes the official plugin supported by RevenueCat.
As a part of this transition, the plugin's npm package has moved from `@capgo/capacitor-purchases` to [`@revenuecat/purchases-capacitor`](https://www.npmjs.com/package/@revenuecat/purchases-capacitor).

## A Heartfelt Thank You to @riderx and @cap-go
We want to extend our deepest gratitude to @riderx and @cap-go for their incredible work on Capacitor Purchases.
Their dedication and skill have laid a strong foundation for the plugin's future.
We are committed to continuing the legacy of excellence they have established.

## What's New in This Release
With this transition, we are also excited to introduce new features and improvements that enhance the plugin's performance, usability, and functionality:

- This release updates the SDK to use BillingClient 5 in Android. This version of BillingClient brings an entire new subscription model which has resulted in large changes across the entire SDK.
- We are adding new functionality previously not available in this plugin to make it as powerful as our other SDKs. Some of these features include:
  - Use `getProducts` and `purchaseStoreProduct` if you're not using RevenueCat offerings system to still allow purchases
  - Set attributes in our SDK so they are available in RevenueCat and any integrations that you use. For example `setEmail`, `setAppsflyerID`, `setFirebaseAppInstanceID`,...
  - Begin a refund request from the SDK in iOS
  - Promotional purchases in iOS
  - And much more.
- Please see a list of our new APIs in [our v6 migration doc](https://github.com/RevenueCat/purchases-capacitor/blob/release/6.0.0/migrations/v6-MIGRATION.md#new-plugin-methods

## Future Roadmap
As the new stewards of Capacitor Purchases, we have big plans for its continued development and improvement. Stay tuned for regular updates, new features, and top-notch support.

## Migration Guides
- See the full list of API changes in [our v6 migration doc](https://github.com/RevenueCat/purchases-capacitor/blob/release/6.0.0/migrations/v6-MIGRATION.md)

- See [Android Native - 5.x to 6.x Migration](https://www.revenuecat.com/docs/android-native-5x-to-6x-migration) for a
  more thorough explanation of the new Google subscription model announced with BillingClient 5 and how to take
  advantage of it in V6. This guide includes tips on product setup with the new model.

### CODE REVIEW

**Comment by tonidero** [comment]
`make-release` failed but I finished it manually.

**Comment by aboedo** [comment]
the link to the migration doc is broken?

**Comment by tonidero** [comment]
> the link to the migration doc is broken?

Oh interesting.. the one in the `CHANGELOG.md` does work, but the one on the PR doesn't. I used a relative path so seems like in the PR description it's not actually pointing to a reference, so it's failing there. Will fix the PR description link

**Review by vegaro — APPROVED** [review]

File: CHANGELOG.latest.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+**RevenueCat Capacitor SDK v6** is here!! 😻
+
+This is the first release of this SDK after RevenueCat took over development. We have moved the plugin to:
+```
+@revenuecat/purchases-capacitor
+```
+
+This release updates the SDK to use BillingClient 5 in Android. This version of BillingClient brings an entire new subscription model which has resulted in large changes across the entire SDK.
+
+Additionally, we have added a lot of new functionality previously not supported on this plugin.
```
[the remark this answers is not in the document]
Comment (aboedo) [reply CHANGELOG.latest.md:10]: Also, I think there's room for improvement here in explaining what happened and giving a shoutout to the previous developer. 

Something like:

```

## Important Announcement: Official Ownership Transition
We are thrilled to announce that RevenueCat has officially taken over the ownership and development of this plugin from @riderx.
This transition marks the beginning of a new chapter for Capacitor Purchases, as it becomes the official plugin supported by RevenueCat.
As a part of this transition, the plugin's npm package has moved from `@capgo/capacitor-purchases` to [`@revenuecat/purchases-capacitor`](https://www.npmjs.com/package/@revenuecat/purchases-capacitor).

## A Heartfelt Thank You to @riderx and @cap-go
We want to extend our deepest gratitude to @riderx and @cap-go for their incredible work on Capacitor Purchases.
Their dedication and skill have laid a strong foundation for the plugin's future.
We are committed to continuing the legacy of excellence they have established.

## What's New in This Release
With this transition, we are also excited to introduce new features and improvements that enhance the plugin's performance, usability, and functionality:

Feature ...: 
Feature ...:


## Future Roadmap
As the new stewards of Capacitor Purchases, we have big plans for its continued development and improvement. Stay tuned for regular updates, new features, and top-notch support.


```

[the remark this answers is not in the document]
Comment (tonidero) [reply CHANGELOG.latest.md]: Thanks @aboedo. That does indeed look much better. I've applied your suggested changelog and added some of the features that have been added with this new version. Lmk what you think!

[the remark this answers is not in the document]
Comment (tonidero) [reply CHANGELOG.latest.md]: Will merge this PR for now but can update the Changelog later as needed.

**Pull request merged**